### PR TITLE
Fixed spelling mistakes.

### PR DESCRIPTION
**Pull request by m1cm1c**

### CODE REVIEW

File: doc/Requirements Specification/Parts/Test Cases.lyx [hunk 1]
```diff
@@ -357,7 +357,7 @@ name "/T70/"
 \end_inset
 
  Assert that Beagle stops measurements after timeout.
- Therefore run a measurment for software (which e.g.
+ Therefore run a measurement for software (which e.g.
  does not terminate) and define an timeout.
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Test Cases.lyx]: “an” -> “a”

Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Test Cases.lyx]: Fixed it.

**Comment by flyfl** [comment]
lgtm

**Comment by jGleitz** [comment]
lgtm

File: doc/Requirements Specification/Parts/Environment.lyx [hunk 2]
```diff
@@ -222,7 +222,7 @@ name "/E40/"
 
 \end_inset
 
- The user should not run other programms while Beagle is running as this
+ The user should not run other programs while Beagle is running as this
```
Comment (acBerger) [inline doc/Requirements Specification/Parts/Environment.lyx]: it is "programms" for BE

Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Environment.lyx]: Oops, sorry. I wasn't aware of this British spelling rule. I'll fix it later.

**Pull request merged**